### ISSUE TITLE
Feature or Clarification

### ISSUE THREAD

**Issue by Red7Hunter** [nicolassturmel/aes67-web-monitor#3]
Hello!

I've been looking around for a pi4 AES67 audio player..

Is there an ability when you select the AES67 stream on the webpage remotely from a client that the sound plays out of the host?

I was trying to get this to get running on raspberrian and npm has a bunch of issues.. I'll try again later on a real pi4 instead of a VM.

I do have it working on pi4 running ubuntu but thats way more overhead than I need.

Thanks,
Red

**Comment by dewiweb** [comment]
Hello @Red7Hunter ,
If you want to monitor individual AES67 streams, maybe it's more convenient to use something dedicated to pi hardware like: https://github.com/martim01/pam

**Comment by Red7Hunter** [comment]
Yes I've looked into that but its way more than I really need.   I really looking just to remotely pick a stream on a pi and have it play it out of its audio device.

Your implementation is very close to this.. didn't know if it had that functionality.

**Comment by dewiweb** [comment]
This app looks promising to monitor AES67 streams: https://aes67.app/
And it comes with a "listen" button by stream
